### PR TITLE
PSP-8586 : FT-REG: Subdivision/ Consolidation - Menu icon enlarges when user clicks on expand the left menu section

### PR DESCRIPTION
**Pull request by eddherrera**

### CODE REVIEW

File: source/frontend/src/components/layout/SideNavBar/styles.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const SideNavBar = styled.div`
   align-items: center;
   grid-area: iconbar;
   background-color: ${props => props.theme.tenant.colour};
-  transition: 0.5s width;
+  transition: 0.3s width;
```
Comment (devinleighsmith) [inline source/frontend/src/components/layout/SideNavBar/styles.ts:23]: what is the motivation for this change?

**Review by devinleighsmith — CHANGES_REQUESTED** [review]
Hmm, I think something is borked in that svg now, prior to this change you can view the svg in the browser as it is considered "valid". see https://github.com/bcgov/PSP/blob/dev/source/frontend/src/assets/images/subdivisionconsolidation.svg

**Comment by devinleighsmith** [comment]
@eddherrera see: https://raw.githubusercontent.com/bcgov/PSP/b42a01ff309fa7952b17ef556975cbede6f64cc6/source/frontend/src/assets/images/subdivisionconsolidation.svg

File: source/frontend/src/components/layout/SideNavBar/styles.ts (hunk 1 above)
Comment (eddherrera) [reply source/frontend/src/components/layout/SideNavBar/styles.ts:23]: I was trying to reduce the animation time, but I see that it's not needed.

**Comment by devinleighsmith** [comment]
@eddherrera looks close, vertical padding is a bit off:

image

**Review by devinleighsmith — APPROVED** [review]
